Servers/Administration/OJS.Servers.Administration/Controllers/BaseAutoCrudAdminController.cs

```diff
@@ -22,4 +24,27 @@ protected override IEnumerable<string> HiddenFormControlNamesOnCreate
         {
             nameof(IEntity<object>.Id),
         };
+
+    protected int GetEntityIdFromQuery(IDictionary<string, string> complexId)
+    {
+        var idString = this.GetPrimaryKeyValueFromQuery(complexId);
+
+        if (!int.TryParse(idString, out var id))
```

Comment:
Why `int`? What if the primary key is of type `Guid`?
Better to provide the `TEntityId` as a generic.